### ISSUE TITLE
do you plan to add extra-trusted-public-keys to the binary caches?

### ISSUE THREAD

**Issue by osbm** [BatteredBunny/nix-ai-stuff#8]
I am sorry if this is intentional. But there is no entry for nixConfig.extra-trusted-public-keys in the flake so we cant pull from the binary caches. Also AFAIK the cuda-maintainers cache was discontinued in favor of nix-community cache [source](https://github.com/SomeoneSerge/nixpkgs-cuda-ci).

**Comment by BatteredBunny** [comment]
Yeah both of these should be fixed, i have all of them also added to my own system binary caches so i never noticed i guess

**Comment by BatteredBunny** [comment]
Should be fixed now
